ello/ellobor_site
# Add in Segment.io analytics

#### What's this PR do?

Adds in Segment analytics.js (also Google Analytics since that project in Segment has it turned on). 
Just added it to the application.js manifest (versus a partial). I also just hardcoded the Segment key in for now.
#### Where should the reviewer start?

With the additional .js files. 
#### How should this be manually tested?
1. Open up Segment, go to the BoR project in there and click the debugger.
2. Open up the site, click one of the social share buttons.
3. Watch the debugger, you should see the initial page load and the share action.
4. If you open the browser inspector/Network tab, you should see two requests, one to Segment and one to GA.
#### Any background context you want to provide?

We say that **you can turn off tracking** on the BoR site but don't actually allow it there (not sure how we would?). If we want that we should probably put this into a partial instead.
#### Screenshots (if appropriate)

n/a
#### Questions:
- Do migrations need to be ran? **no**
- Are there any additional deployment instructions/requirements? **No**, Segment/Google Analytics is already setup.
